### ISSUE TITLE
not able to throttle udate downloading

### ISSUE THREAD

**Issue by YourCompanionCube** [remyroy/CDDA-Game-Launcher#420]
as someone with rather bad internet (since that's all that is available where i live) and multiple people in the house i would like to be able to throttle the maximum download rate of game updates since that way i don't eat all of the network for a few seconds (and spike the ping of someone else trying to play a game say) and can get my update over the course of a few minutes, i'm sure there are more people out there with low end internet like me who this would also let them do other things while the game updates in the background

**Comment by remyroy** [comment]
Good idea. We'll see how we can implement this. This should be achievable with [QNetworkReply::setReadBufferSize](https://doc.qt.io/qt-5/qnetworkreply.html#setReadBufferSize)

**Comment by remyroy** [comment]
You could also use a specialized application like https://www.netlimiter.com/ to do this.
